product/views.py

```diff
@@ -35,3 +35,77 @@ def get(self, request, product_id):
         except Product.DoesNotExist:
             return JsonResponse({'message': 'PRODUCT_NOT_FOUND'}, status = 404)
         return JsonResponse({'message': 'SUCCESS', 'product_detail': product_detail}, status = 200)
+
+class ProductListView(View):
+    def get(self, request):
+        try:
+            offset              = int(request.GET.get('offset', 0))
+            limit               = int(request.GET.get('limit', 4))
+            subcategory_num     = request.GET.get('subcategory', None)
```

Comment:
```suggestion
            subcategory_number     = request.GET.get('subcategory', None)
```
num을 보면 누구나 number임을 유추할 수 있지만, 그래도 최대한 축약어를 줄이시는 것이 좋습니다 :) 혹시 오해할 수 있기 때문입니다.
혹은 `subcategory_id`는 어떠신가요?

Reply:
축약어를 지양하는 위코드 컨벤션을 까먹었었군요😱
`subcategory_id`로 수정하였습니다!